testsuite/ada_lsp/completion.invisible.runtime/test.yaml

```diff
@@ -0,0 +1,3 @@
+title: 'completion.invisible.runtime'
+skip:
+    - ['SKIP', 'env.build.os.name != "linux"']
```

Comment:
could you add a comment? (should we skip this on Mac instead?)

Reply:
I want to test on only one platform because runtime source locations may differ between each of them. I can add a comment saying that.